app/assets/images/payment_icons/alipay_hk.svg

```diff
@@ -0,0 +1 @@
f"/></mask><g mask="url(#pi-alipay_hk-mask2)"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.024 13.96h1.909V9.371h-1.909v4.59z" fill="#393632"/></g><path fill-rule="evenodd" clip-rule="evenodd" d="M15.272 9.753a1.902 1.902 0 00-.476.275v-.304h-.977a.239.239 0 00-.164.067.225.225 0 00-.069.16v5.185h1.21v-1.557c.11.044.222.07.338.084.115.01.252.017.41.017a2.1 2.1 0 00.848-.165c.252-.112.466-.265.644-.463a2.04 2.04 0 00.41-.698c.094-.27.141-.563.141-.882 0-.248-.04-.483-.12-.703a1.774 1.774 0 00-.342-.581 1.601 1.601 0 00-1.225-.543c-.24 0-.449.035-.628.108zm-.033 3.4a.704.704 0 01-.237-.144.646.646 0 01-.206-.483v-1.712c.05-.065.104-.127.163-.187.06-.06.122-.11.188-.154a.913.913 0 01.205-.103.67.67 0 01.222-.038c.106 0 .203.032.293.096.088.065.165.16.229.288.064.128.114.287.15.476.036.189.053.41.053.66 0 .196-.02.377-.06.544a1.65 1.65 0 01-.163.428.87.87 0 01-.239.28.486.486 0 01-.292.103.864.864 0 01-.306-.053z" fill="#393632"/><mask id="pi-alipay_hk-mask3" mask-type="alpha" maskUnits="userSpaceOnUse" x="18" y="9" width="4" height="5"><path d="M18.402 9.831c.032.098.076.271.107.372 1.58-.275 1.884-.062 1.884.606v.21c-.288-.02-.628-.069-.907-.073-1.08-.017-1.472.825-1.472 1.357 0 .93.904 1.791 2.379 1.155v.148h1.234v-2.835c0-.55-.485-.882-.919-1.003a3.5 3.5 0 00-.951-.125 5.5 5.5 0 00-1.355.188zm.586 2.49c0-.45.375-.794.835-.828.291-.021.467.03.57.094v1.331a.84.84 0 01-.57.222.826.826 0 01-.835-.818z" fill="#fff"/></mask><g mask="url(#pi-alipay_hk-mask3)"><path fill-rule="evenodd" clip-rule="evenodd" d="M17.662 14.03h4.317V9.29h-4.317v4.74z" fill="#393632"/></g><path fill-rule="evenodd" clip-rule="evenodd" d="M25.767 9.652c-.05 0-.1.013-.146.04a.268.268 0 00-.099.114l-1.036 2.243-1.258-2.397h-1.277l1.94 3.678-.141.311a14.9 14.9 0 01-.095.197c-.035.074-.09.183-.142.24-.236.259-.773.253-.934.253v.653c.214 0 1.384.166 1.866-1.01l2.003-4.322h-.681z" fill="#393632"/><path fill-rule="evenodd" clip-rule="evenodd" d="M12.624 8.574c0 .317-.29.574-.65.574-.358 0-.649-.257-.649-.574 0-.317.29-.574.65-.574.359 0 .65.257.65.574z" fill="#00A0E8"/><mask id="pi-alipay_hk-mask4" mask-type="alpha" maskUnits="userSpaceOnUse" x="11" y="8" width="2" height="2"><path d="M11.325 8.574c0 .317.29.574.65.574.359 0 .65-.257.65-.574 0-.317-.291-.574-.65-.574-.36 0-.65.257-.65.574z" fill="#fff"/></mask><g mask="url(#pi-alipay_hk-mask4)" fill-rule="evenodd" clip-rule="evenodd" fill="#00A0E8"><path d="M-11.218 20.295h40.965V2.175h-40.965v18.12z"/><path d="M10.973 9.5h2.004V7.648h-2.004v1.854z"/></g><path fill-rule="evenodd" clip-rule="evenodd" d="M30.692 13.65h-.73a.08.08 0 01-.08-.082v-1.505a.08.08 0 00-.08-.08h-1.324v1.666h-.729a.08.08 0 01-.08-.08V9.748a.08.08 0 01.08-.081h.73v1.627h1.323a.08.08 0 00.08-.08V9.748a.08.08 0 01.08-.081h.73v3.982zm4.274 0h-.885a.08.08 0 01-.07-.041l-1.028-1.772-.53.615v1.197h-.718a.08.08 0 01-.08-.08V9.748a.08.08 0 01.08-.081h.719v1.622a.04.04 0 00.071.026l1.348-1.62a.08.08 0 01.062-.028h.933l-1.356 1.583 1.454 2.4z" fill="#00A0E8"/></svg>
```

Comment:
I feel this could be improved because I noticed that amount of `<mask>` added to mask the letter shape. Seeing other icons with letters don't rely on mask, instead they merge the shapes together (probably using "Pathfinder" mode in illustration tool)

I'm saying this, but I don't know how to do it. The last time I used Adobe Illustrator was 10 years ago 😅

Reply:
Ah yeah, would be nice to squash those if possible but unfortunately I have no idea how to and use Figma for SVG editing. I suppose we can stick with this for now. 😅